apps/regression/base.universal.i18n

```diff
@@ -1,3 +1,4 @@
+AffineRegressionFormula  = " y=a·x "
```

Comment:
This is the LinearRegressionFormula!

`LinearRegressionFormula  = " y=a·x+b "` is defined in `apps/shared.universal.i18n` but should be renamed AffineRegressionFormula there and in `apps/graph/graph/banner_view.cpp`, and then used in the affine model.

Reply:
Yes the switch between affine and linear was a bit chaotic